i18n/ar.toml

```
@@ -1,80 +1,91 @@
 # UI strings. Buttons and similar.
 
 [ui_pager_prev]
-other = "السابق"
+other = "السَّابق" 
```

Removing trailing blank space in **this and all other lines**, for example:

```suggestion
other = "السَّابق"
```